pkg/asset/installconfig/gcp/client.go

```diff
@@ -368,6 +370,34 @@ func (c *Client) getServiceUsageService(ctx context.Context) (*serviceusage.Serv
 	return svc, nil
 }
 
+// GetServiceAccounts retrieves a list of service accounts that are valid for a project.
+func (c *Client) GetServiceAccounts(ctx context.Context, project string) ([]string, error) {
+	ctx, cancel := context.WithTimeout(ctx, 1*time.Minute)
+	defer cancel()
+
+	svc, err := iam.NewService(ctx)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed create IAM service")
+	}
+
+	rsp, err := svc.Projects.ServiceAccounts.List(fmt.Sprintf("projects/%s", project)).Do()
```

Comment:
I would think we should call `ServiceAccounts.Get` instead of `List`. I think in some of these cases the number of service accounts could be quite large. I think `Get` would cover our use case?